### ISSUE TITLE
Can't minify on CentOS7 (minify.js missing on npm)

### ISSUE THREAD

**Issue by soichih** [digitalbazaar/forge#475]
I just updated node-forge to the latest version (0.6.47), and I can no longer run "npm run minify". This used to work on previous version..

```
~/git/sca-wf/ui/node_modules/node-forge(master*) $ npm run minify

> node-forge@0.6.47 minify /usr/local/git/sca-wf/ui/node_modules/node-forge
> r.js -o minify.js

Error: Error: ENOENT: no such file or directory, lstat '/usr/local/git/sca-wf/ui/node_modules/node-forge/minify.js'
    at Error (native)


npm ERR! Linux 3.10.0-514.2.2.el7.x86_64
npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "run" "minify"
npm ERR! node v6.9.1
npm ERR! npm  v3.10.8
npm ERR! code ELIFECYCLE
npm ERR! node-forge@0.6.47 minify: `r.js -o minify.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the node-forge@0.6.47 minify script 'r.js -o minify.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the node-forge package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     r.js -o minify.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs node-forge
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls node-forge
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /usr/local/git/sca-wf/ui/node_modules/node-forge/npm-debug.log

```

I saw similar issue reported for Windows, but none of the workaround applied, or fixed this issue.  Help!

**Comment by davidlehn** [comment]
I got aggressive removing cruft.  Added back minify script and support files.  Released as 0.6.48.  Let us know if anything else is missing.
https://github.com/digitalbazaar/forge/tree/0.6.x

**Comment by soichih** [comment]
Thanks. It worked this time.
